apps/web-twig-demo/src/Controller/HelpersController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class HelpersController extends AbstractController
+{
+    #[Route('/helpers')]
+    public function index(): Response
+    {
+        return $this->render('helpers.html.twig', ['helpers' => $this->getWebTwigHelpers()]);
+    }
+
+    /**
+     * @return Array<string>
+     */
+    private function getWebTwigHelpers(): array
```

Comment:
Could be shared with `DefaultController` through an abstract controller maybe…? @literat

Reply:
Yes, I think it can be shared via AbstractController.